velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -731,6 +731,12 @@ class PlanBuilder {
       const std::vector<std::string>& unnestColumns,
       const std::optional<std::string>& ordinalColumn = std::nullopt);
 
+  /// Create WindowNode based on whether input is sorted and then compute the
+  /// window functions.
+  PlanBuilder& commonWindow(
```

Comment:
This should be a private method. We usually don't use names like commonWindow. Maybe rename to 'window'.

Reply:
Updated.